tests/integration/components/event-table-test.js

```diff
@@ -0,0 +1,12 @@
+import { moduleForComponent, test } from 'ember-qunit';
+import hbs from 'htmlbars-inline-precompile';
+
+moduleForComponent('event-table', 'Integration | Component | event table', {
```

Comment:
@niranjan94 I have used the same test as was used on navbar component and profile page component, which test should this component test be similar to?